jobs/send-stale-initial-pr-reminder.js

```diff
@@ -11,13 +12,24 @@ module.exports = async function (
   repositoryId = String(repositoryId)
 
   const { installations, repositories } = await dbs()
+  const logs = dbs.getLogsDB()
+  const log = Log({logsDb: logs, accountId, repoSlug: repodoc.fullName, context: 'send-stale-initial-pr-reminder'})
   const installation = await installations.get(accountId)
   const repository = await repositories.get(repositoryId)
   const installationId = installation.installation
   const ghqueue = githubQueue(installationId)
 
-  if (repository.enabled) return
-  if (repository.staleInitialPRReminder) return
+  log.info('started')
+
+  if (repository.enabled) {
+    log.info('stopped: repository disabled')
```

Comment:
Message is inverted, the job stops _because_ the repo is enabled, then we don’t need to send a stale initial PR reminder.